### DISCUSSION TITLE
AHK++ launch config question

### DISCUSSION THREAD

**Discussion by kyklish** [mark-wiemer/ahkpp#428] [General]
@mark-wiemer  Hardcoded `AutoHotkeyU64.exe` version in two places. Why?

https://github.com/mark-wiemer-org/ahkpp/blob/14b1114342aaa89a668100b46ca897d41a0ce0e5/.vscode/launch.json#L21
https://github.com/mark-wiemer-org/ahkpp/blob/14b1114342aaa89a668100b46ca897d41a0ce0e5/demos/.vscode/launch.json#L21

**Comment by kyklish** [comment]
`package.json`
```json
    "keywords": [
        "vscode",
        "autohotkey",
        "ahk",
        "ahk2",
        "ahk v2",
        "ahk v1"    <---- move up one line (higher then "ahk v2") ???
    ],
```

**Reply by mark-wiemer** [reply]
I don't think the keyword order matters too much, though you're welcome to open a PR :)

**Comment by mark-wiemer** [comment]
The demos project is meant to be an example for other people to use in their project. The main ahkpp/.vscode/launch.json is the file actually used by this project. For now, they're the same, but they may change in the future if this project adds advanced debug options that aren't needed to get started.
